docs/troubleshooting.md

```diff
@@ -78,7 +78,15 @@ For users who want to deploy to the local cluster as well, they may move the clu
 
 **Result**: The cluster will be migrated to `fleet-default`.
 
-## **Additional Solutions for Other Fleet Errors**
+## **Additional Solutions for Other Fleet Issues**
+
+### Naming conventions for CRDs
+
+1. For CRD terms like `clusters` and `gitrepos`, you must reference the full CRD name. For example, the cluster CRD's complete name is `clusters.fleet.cattle.io`, and the gitrepo CRD's complete name is `gitrepo.fleet.cattle.io`.
```

Comment:
```suggestion
1. For CRD terms like `clusters` and `gitrepos`, you must reference the full CRD name. For example, the cluster CRD's complete name is `cluster.fleet.cattle.io`, and the gitrepo CRD's complete name is `gitrepo.fleet.cattle.io`.
```
For full name singluar or plural are both fine, we can probably stick with singular now.